src/view/dataElements/xdmObject/components/sandboxSelector.jsx

```diff
@@ -40,7 +40,7 @@ const SandboxSelector = ({
   const reportAsyncError = useReportAsyncError();
   const isFirstRender = useIsFirstRender();
 
-  const sandboxList = useAsyncList({
+  const sandboxList = useListData({
     async load({ signal }) {
```

Comment:
Will this work? I don't see `load()` on the list of properties for `useListData()`. Instead of `load()`, it takes a static array through the items property `items`.

[When I suggested using `useListData()` in the previous PR](https://github.com/adobe/reactor-extension-alloy/pull/215#discussion_r852410946), it was for a the component in `src/view/configuration/sandboxSelector.jsx`.

I think this should be `useAsyncList()` because `load()` calls an asynchronous function.